camayoc/tests/qcs/api/v1/scans/test_multi_source_scans.py

```diff
@@ -0,0 +1,44 @@
+# coding=utf-8
+"""Tests for ``Scan`` API endpoint for quipucords server.
+
+:caseautomation: automated
+:casecomponent: api
+:caseimportance: high
+:caselevel: integration
+:requirement: Sonar
+:testtype: functional
+:upstream: yes
+"""
+
+import pytest
+
+from camayoc.tests.qcs.api.v1.utils import (
+    prep_all_source_scan,
+    wait_until_state,
+)
+
+
+@pytest.mark.parametrize('scan_type', ['connect', 'inspect'])
+def test_multi_source_create(shared_client, cleanup, scan_type):
+    """Run a scan on a system and confirm it completes.
+
+    :id: 43624cc1-6c41-4c2d-9919-c3d0aae83165
+    :description: Create a scan and assert that it completes
```

Comment:
I think you can expand a little bit more with details here. I can't tell what should go into the scan in order to complete this test.